### DISCUSSION TITLE
How to use OPENAI API

### DISCUSSION THREAD

**Discussion by xuchenCN** [open-webui/open-webui#3963] [Q&A]
I set the OPENAI API and API_KEY, and restart docker container, but nothing changes from webui, is there some option I missed?

I try to select a models in chat page, but no models here.

![image](https://github.com/user-attachments/assets/5718d95a-0ef5-477c-be06-1371c4119b5b)

![image](https://github.com/user-attachments/assets/ade2d85a-4d39-4674-a290-e6ca006abd45)

**Comment by xuchenCN** [comment]
Anyone give me some suggestion ?

**Reply by TrippyToes** [reply]
have you tried restarting openwebui after setting up the API?

**Reply by xuchenCN** [reply]
Yes, config via ENV or in admin panel, and restart. 

Could you please depict the feature how to works in openwebui?

**Comment by xuchenCN** [comment]
OK, the issue is local certificate not working.
